File: src/main/java/me/soo/helloworld/model/language/LanguageData.java
package me.soo.helloworld.model.language;

import lombok.AllArgsConstructor;
import lombok.Getter;
import me.soo.helloworld.enumeration.LanguageLevel;

@Getter
@AllArgsConstructor
public class LanguageData {

    private final int id;

    private final LanguageLevel level;

}

File: src/main/java/me/soo/helloworld/model/language/LanguageData.kt
package me.soo.helloworld.model.language

import me.soo.helloworld.enumeration.LanguageLevel
import me.soo.helloworld.enumeration.LanguageStatus

data class LanguageData(val id: Int, val level: LanguageLevel, val status: LanguageStatus)
